sessions/sessions.go

```diff
@@ -20,34 +20,11 @@ func NewClient() *redis.Client {
 	})
 }
 
-func Lock() {
-	fmt.Printf("Waiting for lock\n")
-	for {
-		b, err := client.SetNX("lock", "lock", time.Second*10).Result()
-		fmt.Printf("b,err = %v, %v\n", b, err)
-		if b {
-			break
-		}
-		time.Sleep(time.Second)
-	}
-	fmt.Printf("Got it\n")
-}
-
-func Unlock() {
-	fmt.Printf("Unlocking\n")
-	client.Del("lock")
-	fmt.Printf("Unlocked\n")
-}
-
 func Handler(w http.ResponseWriter, r *http.Request) {
-	Lock()
-	count, err := client.Get("count").Int()
-	if err != nil && err.Error() != "redis: nil" {
```

Comment:
For a brand new Redis setup, I didn't see there is a "redis: nil" error - guess `Incr()` has covered the nil case underneath.